x-pack/platform/plugins/shared/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/hyper_parameters.tsx

```diff
@@ -176,8 +179,7 @@ export const HyperParameters: FC<Props> = ({ actions, state, advancedParamErrors
               })
             }
             isInvalid={
-              featureBagFraction !== undefined &&
```

Comment:
@elastic/ml-ui, please confirm that the value copied from `EuiFormRow` is relevant